frontend_lib/src/container/PopupUploadFile.jsx

```diff
@@ -95,6 +95,12 @@ class PopupUploadFile extends React.Component {
   handleValidate = async () => {
     const { state, props } = this
 
+    // INFO - CH - 20210315 - this allows to handle the upload outside of this component
+    if (props.onValidateOverride !== false) {
+      props.onValidateOverride(state.fileUploadList)
+      return
+    }
+
```

Comment:
Couldn't a `handleValidate` props with its default value set to the previous code be used instead? And change it if an override is needed?

Reply:
mmh interesting idea, I'll look into it

Reply:
I couldn't find a proper way to do it

Reply:
won't fix